debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/ObjectFieldInfo.java

```diff
@@ -71,10 +71,11 @@
 	int totalFlatFieldRefBytes = 0;
 	int totalFlatFieldSingleBytes = 0;
 
-	public static final int	NO_BACKFILL_AVAILABLE = -1;
-	public static final int		BACKFILL_SIZE = U32.SIZEOF;
-	public static final int		LOCKWORD_SIZE = j9objectmonitor_t_SizeOf;
-	public static final int		FINALIZE_LINK_SIZE = fj9object_t_SizeOf;
+	public static final int NO_BACKFILL_AVAILABLE = -1;
+	public static final int BACKFILL_SIZE = U32.SIZEOF;
+	public static final int LOCKWORD_SIZE = j9objectmonitor_t_SizeOf;
+	public static final int FINALIZE_LINK_SIZE = fj9object_t_SizeOf;
+	public static final int OBJECT_SIZE_INCREMENT_IN_BYTES = U64.SIZEOF;
```

Comment:
Why not use `ObjectFieldInfo.OBJECT_SIZE_INCREMENT_IN_BYTES` instead of defining this field?